build/root/var/lib/ci-artifacts/entrypoints.d/sro_run_libs.sh

```diff
@@ -0,0 +1,48 @@
+prepare_cluster_for_sro() {
+    toolbox/cluster/capture_environment.sh
+
+    entitle
+
+    if ! toolbox/nfd/has_nfd_labels.sh; then
+        toolbox/nfd/deploy_from_operatorhub.sh
+    fi
+}
+
+validate_sro_deployment() {
+    trap toolbox/special-resource-operator/capture_deployment_state.sh ERR
+
+    toolbox/special-resource-operator/run-e2e-test.sh "${CI_IMAGE_SRO_COMMIT_CI_REPO}" "${CI_IMAGE_SRO_COMMIT_CI_REF}"
+    toolbox/gpu-operator/capture_deployment_state.sh
```

Comment:
you need to call `trap - ERR` before calling `toolbox/gpu-operator/capture_deployment_state.sh`
otherwise if `capture` fails, you'll call `capture` again

see how changed it there:
https://github.com/openshift-psap/ci-artifacts/pull/111/files#diff-5ac85e504d9fba79fc112fb78c18befbcfad19a10bffe0a5ebc963ba79154778L151

... and copy and paste error, you want to call `toolbox/special-resource-operator/capture_deployment_state.sh` at the end!